.mk/ocp.mk

```diff
@@ -8,6 +8,9 @@ ocp-expose:
 	oc expose service loki || true
 	@loki_url=$$(oc get route loki -o jsonpath='{.spec.host}'); \
 	echo -e "\nAccess loki on OCP using: http://"$$loki_url"\n"
+	oc expose -n sample-workload service frontend-external || true
```

Comment:
I tend to use `make ocp-expose` just to create a grafana route. Can you split in two different targets then? (maybe `ocp-expose` and `ocp-expose-all`, similarly to the `deploy-all` that deploys more than just the infra ?

Reply:
Fixed (@jotak  please review again that this fits your needs )